README.md

```diff
@@ -35,6 +35,10 @@ For automatic running of linters before you commit:
 $ pip install pre-commit
 $ pre-commit install
 ```
+
+### Contribution 
+Want to contribute to the platform ? Checkout the [contribution guidelines](https://github.com/fossunited/fossunited/blob/develop/CONTRIBUTING.md). 
```

Comment:
nitpicky comment - if i remember correctly, you should be able to link to a local markdown file from a markdown file i.e. `./CONTRIBUTING.md` instead of putting the GitHub URL in the link. Ref https://www.markdownguide.org/basic-syntax/#links where they mention this for images.

Reply:
see also https://github.github.com/gfm/ for the GitHub flavor Markdown references

Reply:
> nitpicky comment - if i remember correctly, you should be able to link to a local markdown file from a markdown file i.e. `./CONTRIBUTING.md` instead of putting the GitHub URL in the link. Ref https://www.markdownguide.org/basic-syntax/#links where they mention this for images.

Oh yeah, makes sense. Will change that